SmartDeviceLink/SDLIAPTransport.m

```diff
@@ -117,9 +124,13 @@ - (void)sdl_applicationWillEnterForeground:(NSNotification *)notification {
 #pragma mark - Stream Lifecycle
 
 - (void)connect {
+    [self sdl_connect:nil];
+}
+
+- (void)sdl_connect:(EAAccessory *)accessory {
```

Comment:
That's not correct. We call sdl_connect: with the accessory provided in sdl_accessoryConnected.